Resources/Prototypes/Catalog/Fills/Crates/armory.yml

```diff
@@ -74,7 +74,7 @@
   id: CrateSecurityRiot
   parent: [ CrateWeaponSecure, BaseRestrictedContraband ]
   name: swat crate
-  description: Contains two sets of riot armor, helmets, shields, and enforcers loaded with beanbags. Extra ammo is included. Requires Armory access to open.
+  description: Contains two sets of riot armor, helmets, shields, and Stun Enforcers. Extra stun ammo included. Requires Armory access to open.
```

Comment:
i thhhhhink the capitalization on Stun Enforcers is inconsistent? pretty sure stun shouldnt be capitalized? i may be wrong here

Reply:
I'll check elsewhere, thanks!